portal-impl/src/com/liferay/portal/minifier/MinifierUtil.java

```diff
@@ -50,6 +62,32 @@ public static String minifyJavaScript(String resourceName, String content) {
 		return content;
 	}
 
+	private static String _getServletContextPath(String resourceName) {
+		String pathModule = Portal.PATH_MODULE;
+
+		int i = resourceName.indexOf(StringPool.SLASH, pathModule.length() + 1);
+
+		if (i == -1) {
+			return StringPool.BLANK;
+		}
+
+		return resourceName.substring(0, i);
+	}
+
+	private static boolean _isLiferayResource(String resourceName) {
+		if (resourceName.startsWith(
+				Portal.PATH_MODULE + "/js/resolved-module/") ||
```

Comment:
would this catch the third party js widgets providing modules?

Reply:
Mmm, yes, but maybe we can try to ship it like this, since all these modules should be routed through the bundler so we can minify them in that point, instead of here (if someone complains).

Right now I'm not totally sure of what we are doing in the bundles. I think we leave the code it as it comes, but I will check if we need minification.

Reply:
https://github.com/liferay/liferay-frontend-projects/issues/976